src/custom/ErrorBoundary/ErrorBoundary.tsx

```diff
@@ -30,14 +32,25 @@ const CodeMessage = styled('div')(({ theme }) => ({
 interface FallbackComponentProps extends FallbackProps {
   resetErrorBoundary: () => void;
   children?: React.ReactNode;
+  pageUrl?: string;
+  timestamp?: string;
 }
 
 export function Fallback({ error, children }: FallbackComponentProps): JSX.Element {
+  const pageUrl = window.location.href;
+  const timestamp = new Date().toLocaleString();
   return (
     <div role="alert">
       <h2>Uh-oh!😔 Please pardon the mesh.</h2>
       <CodeMessage>
-        <code>{(error as Error).message}</code>
+        <code>
+          <strong>Error: </strong>
+          {(error as Error).message}
+        </code>
+        <br />
+        <strong> URL: </strong> {pageUrl}
+        <br />
+        <strong>Logged at:</strong> {timestamp}
```

Comment:
@sudhanshutech thanks for this, however, this information is helpful for contributors and maintainers to be aware of. I wonder if it helps to show this to users.

Reply:
I thinks it can be helpful, like if any user want to report any issue they had so they can report the exact url and time or the version they were using.

Reply:
because just showing them a simple error in there face in case any bug occurs feels the information is incomplete.

Reply:
ok i am just showing system version now and other details will be just available inside feedback not inside error boundary